### ISSUE TITLE
Is it possible to use this with 32-bit builds of Prepar3D?

### ISSUE THREAD

**Issue by JohnnyCurran** [Sequal32/simconnect-rust#5]
64-bit versions of prepar3d are backwards compatible with 32-bit versions. A 32-bit version written with the help of this library would then support all versions of prepar3d.

I gave it a shot with 32-bit SimConnect.dll's and .lib's while targeting `stable-i686-pc-windows-gnu`  but I can't get it to compile.

Any tips? The library as it stands returns `E_FAIL` from `SimConnect_GetNextDispatch`

**Comment by JohnnyCurran** [comment]
Seems like it'd be possible if you could specify the toolset version when targeting `stable-i686-pc-windows-msvc`. For P3Dv3 specifically it requires Visual Studio 2013 (v120) toolset in order for the linker not to throw a fit.

**Comment by Sequal32** [comment]
Yeah I've tried as well but can't get the linker to find the symbols in the lib file for FSX:SE... it might as well be as you've stated that we need an older toolset.
